vars/dotenv2yaml.groovy

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env groovy
+
+void call(String envfile, String outfile) {
+  sh """
+    set -o errexit
+    set -o nounset
+    set -o pipefail
+
+    cat ${envfile} | docker run --rm -i gcr.io/mindmixer-sidewalk/dotenv-to-yaml > ${outfile}
```

Comment:
>gcr.io/mindmixer-sidewalk/dotenv-to-yaml

My only note here is that we'll need to add the Dockerfile that built this image to the dev-ops repo.

Reply:
I snuck the `Dockerfile` into [this PR](https://github.com/mysidewalk/dev-ops/pull/658)